pkg/commands/drift/github.go

```diff
@@ -17,92 +17,70 @@ package drift
 import (
 	"context"
 	"fmt"
-	"strings"
 
 	"github.com/abcxyz/guardian/pkg/github"
 	githubAPI "github.com/google/go-github/v53/github"
 )
 
-const (
-	issueTitle = "IAM drift detected"
-	issueBody  = `We've detected a drift between your submitted IAM policies and actual
-        IAM policies.
-
-        See the comment(s) below to see details of the drift
-
-        Please determine which parts are correct, and submit updated
-        terraform config and/or remove the extra policies.
-
-        Re-run drift detection manually once complete to verify all diffs are properly resolved.`
-)
+type GitHubDriftIssueService struct {
+	GH         *github.GitHubClient
```

Comment:
Style nit: it'd be better to name this GitHubClient or just Client. Also, does it need to be public?

Reply:
Will update the name. Regarding it being public - I want to use the Service outside of the drift pkg (in statefiles). Precedent would lead me to believe that we should create a constructor. In this case I thought adding the constructor only adds an unnecessary function + call to function (one with a lot of string arguments). wdyt?

Reply:
I believe he is referring to `GH` -> `githubClient` so its private to the struct, not the GitHubDriftIssueService. I think its fine that GitHubDriftIssueService is public and reusable. If its common enough can we move it into the `github` package instead (or make it generic enough to move it in there)?

Reply:
Right right. But I don't have a constructor for the  GitHubDriftIssueService. So in order to create an instance of it you need to set the necessary private member variables (doesn't work from another pkg if they are private).

Reply:
Done